modules/NuGetPackageVerifier/console/Rules/PackageOwnershipRule.cs

```diff
@@ -3,127 +3,199 @@
-                        }
-                    }
-                }
+        };
 
-                if (!AllowedOwners.Intersect(owners, StringComparer.OrdinalIgnoreCase).Any())
-                {
-                    yield return PackageIssueFactory.IdIsNotOwned(context.Metadata.Id, AllowedOwners);
-                }
-            }
-        }
+        // Packages that start with one of the reserved prefixes but are not owned by Microsoft or ASP.Net.
+        private static readonly string[] ExternallyOwnedPackageIds = new string[]
+        {
+            "Microsoft.AspNet.Authentication.QQ",
+            "Microsoft.AspNet.Authentication.WeChat",
+            "Microsoft.AspNet.Diagnostics.StatusCodePagesOverrides",
+            "Microsoft.AspNet.HealthChecks",
+            "Microsoft.AspNet.Identity.EntityFramework6",
+            "Microsoft.AspNet.Identity.Guid",
+            "Microsoft.AspNet.Identity.Guid.pt-BR",
+            "Microsoft.AspNet.Mvc-vHalfNext",
+            "Microsoft.AspNet.Mvc.Futures-vHalfNext",
+            "Microsoft.AspNet.OData.EntityBuilder",
+            "Microsoft.AspNet.OData.Extensions.ODataQueryMapper",
+            "Microsoft.AspNet.OData.Versioning.ApiExplorer",
+            "Microsoft.AspNet.OData.Versioning",
+            "Microsoft.AspNet.Razor-vHalfNext",
+            "Microsoft.AspNet.SignalR.Client.Portable",
+            "Microsoft.AspNet.SignalR.Ninject",
+            "Microsoft.AspNet.WebApi.Extensions.Compression.Server",
+            "Microsoft.AspNet.WebApi.Extensions.Compression.Server.Owin",
+            "Microsoft.AspNet.WebApi.Extensions.Compression.Server.Owin.StrongName",
+            "Microsoft.AspNet.WebApi.Extensions.Compression.Server.StrongName",
+            "Microsoft.AspNet.WebApi.HelpPage.Ex",
+            "Microsoft.AspNet.WebApi.LocalizedHelpPage",
+            "Microsoft.AspNet.WebApi.MessageHandlers.Compression",
+            "Microsoft.AspNet.WebApi.Versioning.ApiExplorer",
+            "Microsoft.AspNet.WebApi.Versioning",
+            "Microsoft.AspNet.WebApi.WebHost.zh-Hans1",
+            "Microsoft.AspNet.WebPages-vHalfNext",
+            "Microsoft.AspNetCore.ApiHelp.Core",
+            "Microsoft.AspNetCore.ApiHelp",
+            "Microsoft.AspNetCore.ApiWidgets",
+            "Microsoft.AspNetCore.Authentication.ActiveDirectory",
+            "Microsoft.AspNetCore.Authentication.LinkedIn",
+            "Microsoft.AspNetCore.Authentication.QQ",
+            "Microsoft.AspNetCore.Authentication.QQConnect",
+            "Microsoft.AspNetCore.HealthChecks",
+            "Microsoft.AspNetCore.Identity.MongoDB",
+            "Microsoft.AspNetCore.JsonPatch.Net40",
+            "Microsoft.AspNetCore.Mono",
+            "Microsoft.AspNetCore.Mvc.Formatters.Xml.Extensions",
+            "Microsoft.AspNetCore.Mvc.Versioning.ApiExplorer",
+            "Microsoft.AspNetCore.Mvc.Versioning",
+            "Microsoft.AspNetCore.OData",
```

Comment:
Are they aware of this?

Reply:
Dunno. @danroth27 can you get in touch with the OData folks to let them know that some 3rd party has taken up some names that they might otherwise want to use? (Just an FYI, it's up to them if they want to do anything about it.)